superset-frontend/src/visualizations/presets/MainPreset.js

```diff
@@ -134,9 +141,33 @@ export default class MainPreset extends Preset {
         new TreemapChartPlugin().configure({ key: 'treemap' }),
         new WordCloudChartPlugin().configure({ key: 'word_cloud' }),
         new WorldMapChartPlugin().configure({ key: 'world_map' }),
+        new EchartsAreaChartPlugin().configure({
+          key: 'echarts_area',
+        }),
         new EchartsTimeseriesChartPlugin().configure({
           key: 'echarts_timeseries',
         }),
+        new EchartsTimeseriesBarChartPlugin().configure({
+          key: 'echarts_timeseries_bar',
+        }),
+        new EchartsTimeseriesLineChartPlugin().configure({
+          key: 'echarts_timeseries_line',
+        }),
+        new EchartsTimeseriesSmoothLineChartPlugin().configure({
+          key: 'echarts_timeseries_smooth',
+        }),
+        new EchartsTimeseriesScatterChartPlugin().configure({
+          key: 'echarts_timeseries_scatter',
+        }),
+        new EchartsTimeseriesStepStartChartPlugin().configure({
+          key: 'echarts_timeseries_step_start',
+        }),
+        new EchartsTimeseriesStepMiddleChartPlugin().configure({
+          key: 'echarts_timeseries_step_middle',
+        }),
+        new EchartsTimeseriesStepEndChartPlugin().configure({
+          key: 'echarts_timeseries_step_end',
+        }),
```

Comment:
I wonder if these should just collectively be `EchartsTimeseriesStepChartPlugin` with an additional control to define start/middle/end

Reply:
Good question. @junlincc WDYT?